### ISSUE TITLE
Make NFT Collections Bootstrappable by a DAO

### ISSUE THREAD

**Issue by 0xekez** [DA0-DA0/dao-contracts#642]
By the order of the corporation, DAOs should be able to bootstrap NFT collections. To do this:

1. DAOs need to be able to create a NFT collection on instantiation.
2. And, mint NFTs as part of instantiation.

**Comment by 0xekez** [comment]
The steps for this:

1. dao-core instantiates dao-voting-cw721-staked
2. dao-voting-cw721-staked instantiates a cw721-base contract using provided `ModuleInstantiateInfo` and sets itself as the minter on the cw721-base contract
3. on reply for instantiation, dao-voting-cw721-staked mints NFTs (this will require modifying the instantiate message)
4. once NFTs are minted, dao-voting-cw721-staked changes the minter of the cw721-base contract to dao-core

**Comment by 0xekez** [comment]
this will require a modification to the cw721-base contract to allow the minter to be updated.

**Comment by 0xekez** [comment]
blocked on https://github.com/CosmWasm/cw-nfts/pull/109 for updatable minters

**Comment by JakeHartnell** [comment]
Closing as a duplicate of #665

**Comment by JakeHartnell** [comment]
Work for this in #692
